sdk/python/feast/contrib/validation/ge.py

```diff
@@ -0,0 +1,126 @@
+import io
+import json
+from typing import TYPE_CHECKING
+from urllib.parse import urlparse
+
+import pandas as pd
+
+from feast.constants import ConfigOptions
+from feast.contrib.validation.base import serialize_udf
+from feast.staging.storage_client import get_staging_client
+
+try:
+    from great_expectations.core import ExpectationSuite
+    from great_expectations.dataset import PandasDataset
+except ImportError:
+    raise ImportError(
+        "great_expectations must be installed to enable validation functionality. "
+        "Please install feast[validation]"
+    )
+
+try:
+    from pyspark.sql.types import BooleanType
+except ImportError:
+    raise ImportError(
+        "pyspark must be installed to enable validation functionality. "
+        "Please install feast[validation]"
+    )
+
+
+if TYPE_CHECKING:
+    from feast import Client, FeatureTable
+
+
+GE_PACKED_ARCHIVE = "https://storage.googleapis.com/feast-jobs/spark/validation/pylibs-ge-%(platform)s.tar.gz"
```

Comment:
How does this work from distribution perspective? I worry that if this is not in any way tied to Feast version, we can't upgrade GE without breaking older versions of ingestion job.

Maybe we don't have to address this now, especially while this is contrib, but sometime down the road we probably need to pin the version of this tarball to a Feast version somehow.

Reply:
I know that smells. But since feature is experimental I think it's ok for now.
As an option for future - we could put this archive inside ingestion jar.